app/helpers/administrate/application_helper.rb

```diff
@@ -15,6 +15,20 @@ def render_field(field, locals = {})
       render locals: locals, partial: field.to_partial_path
     end
 
+    def requireness(field)
```

Comment:
Shouldn't we have specs for these other methods as well?

Reply:
These two methods, in the end, calls "has_presence_validator?" method. If I test "requireness" method I would be testing the Ruby's ternary operator, no? Do you think it is necessary?

Reply:
You will be testing the return of your function depending on the params passed to it.

Reply:
I think the provided specs are enough. Testing `requireness` already is a test for the others.